nixos/modules/services/continuous-integration/buildkite-agent.nix

```diff
@@ -31,6 +31,14 @@ in
         '';
       };
 
+      hooks-path = mkOption {
```

Comment:
NixOS options are camelCase

Reply:
As you can see, the immediately following option is called `meta-data`.

I erred on the side of following the particular author's naming convention.

Should it be overrided?

Reply:
`meta-data` should also be renamed, but that's out of scope for this PR.

Reply:
@domenkozar, done